src/environment-variables/index.js

```diff
@@ -3,6 +3,7 @@ const dotenv = require('dotenv');
 
 const defaultValues = {
   EPISODE_PATH: '.',
+  EPISODE_CONFIG: 'episode.json',
```

Comment:
In the past we specified only episode path, without the actual episode file.
Do you think it is a better idea to have only one variable for specifying the episode config file?

Maybe the name of that variable could be `EPISODE_CONFIG_PATH` or just the same old `EPISODE_CONFIG`.

Reply:
`EPISODE_PATH` is still there for backwards compatibility. Why suddenly breaking someone's pipeline that's been running smoothly for a year or more?"

I'll totally adopt your suggestion if you want, though.

Reply:
If I were designing this from scratch, I would agree with @matevskial that there is no need for two variables when we could have one that includes the path and file name.

But considering that we have previously implemented the option to customize the `EPISODE_PATH` variable to specify the **path**, changing this behavior would break backwards compatibility.

Therefore, I support @weltonrodrigo's approach of maintaining the original `EPISODE_PATH` variable for the **path** and introducing a new variable, `EPISODE_CONFIG`, to specify the file.

Reply:
The only consideration I have is whether we should use `EPISODE_FILE` to describe the variable for the file instead of `EPISODE_CONFIG`, since we have already chosen the name `EPISODE_PATH` for the variable for the path. 

@weltonrodrigo @matevskial, what are your thoughts on this?

Reply:
I agree with both of you.

`EPISODE_FILE` is a good name.

Reply:
Solved with 68f3bab